modules/gapi/test/common/gapi_video_tests_common.hpp

```diff
@@ -321,6 +321,31 @@ inline GComputation runOCVnGAPIOptFlowPipeline(TestFunctional& testInst,
     return c;
 }
 
+inline void testBackgroundSubtractorStreaming(cv::GStreamingCompiled gapiBackSub,
+                                              cv::Ptr<cv::BackgroundSubtractor> pOCVBackSub,
```

Comment:
```
@OrestChura OrestChura 6 hours ago Member
please, const reference parameters where applicable

@anna-khakimova anna-khakimova 35 minutes ago Author Contributor
This comment for gapiBackSub?
```
Yes, for `gapiBackSub` for sure, but now for `pOCVBackSub`, too, I'm afraid. If it is applicable.
Const is probably not, but reference - maybe. I'm not really sure, please check if `cv::Ptr` can be given as a reference.
All in all, It is a _structure_ which saves a pointer inside.

Reply:
Ok. Done.